--- bessi0.f.orig	2002-10-14 12:40:58.000000000 -0700
+++ bessi0.f	2022-11-08 08:42:51.636940627 -0800
@@ -1,4 +1,5 @@
-      FUNCTION BESSI0(X)
+      REAL*8 FUNCTION BESSI0(X)
+      IMPLICIT REAL*8(A-H,O-Z)
       REAL*8 Y,P1,P2,P3,P4,P5,P6,P7,
      *    Q1,Q2,Q3,Q4,Q5,Q6,Q7,Q8,Q9
       DATA P1,P2,P3,P4,P5,P6,P7/1.0D0,3.5156229D0,3.0899424D0,1.2067492D
@@ -7,13 +8,13 @@
       DATA Q1,Q2,Q3,Q4,Q5,Q6,Q7,Q8,Q9/0.39894228D0,0.1328592D-1,
      *    0.225319D-2,-0.157565D-2,0.916281D-2,-0.2057706D-1,
      *    0.2635537D-1,-0.1647633D-1,0.392377D-2/
-      IF (ABS(X).LT.3.75) THEN
-        Y=(X/3.75)**2
+      IF (DABS(X).LT.3.75D0) THEN
+        Y=(X/3.75D0)**2
         BESSI0=P1+Y*(P2+Y*(P3+Y*(P4+Y*(P5+Y*(P6+Y*P7)))))
       ELSE
-        AX=ABS(X)
-        Y=3.75/AX
-        BESSI0=(EXP(AX)/SQRT(AX))*(Q1+Y*(Q2+Y*(Q3+Y*(Q4
+        AX=DABS(X)
+        Y=3.75D0/AX
+        BESSI0=(DEXP(AX)/DSQRT(AX))*(Q1+Y*(Q2+Y*(Q3+Y*(Q4
      *      +Y*(Q5+Y*(Q6+Y*(Q7+Y*(Q8+Y*Q9))))))))
       ENDIF
       RETURN
